.deny.toml

```diff
@@ -76,6 +74,3 @@ allow-git = [
 unknown-registry = "deny"
 unknown-git = "deny"
 required-git-spec = "rev"
-
-[sources.allow-org]
-github = ["gfx-rs"]
```

Comment:
I have yet to revert this back out. I added `command-arguments: -Dwarnings` to CI to catch some warnings that otherwise went unnoticed, but that includes unmatched `allow-org` when there are (temporarily) no patches from this org...